Microsoft.Toolkit.Uwp.SampleApp/SamplePages/SystemInformation/SystemInformationPage.xaml.cs

```diff
@@ -70,6 +73,16 @@ public sealed partial class SystemInformationPage : Page
         // To get the time the app was previously launched, not including this instance
         public string LastLaunchTime => SystemInformation.LastLaunchTime.ToString(Culture.DateTimeFormat);
 
+        // To get the time the launch count was reset, not including this instance
```

Comment:
Would it not just be easier and clearer to make 2 dependency properties?

Reply:
I don't see. I like the way you have all the read-only properties. I simply added auto-updated properties. Seems pretty straightforward to me.